StructureDefinition/England-HealthcareService.xml

```diff
@@ -0,0 +1,307 @@
+<StructureDefinition xmlns="http://hl7.org/fhir">
+    <id value="England-HealthcareService" />
+    <url value="https://fhir.nhs.uk/StructureDefinition/England-HealthcareService" />
+    <name value="EnglandHealthcareService" />
+	<title value="England HealthcareService" />
+    <status value="draft" />
+    <fhirVersion value="4.0.1" />
+    <kind value="resource" />
+    <abstract value="false" />
+	<version value="1.0.0" />
```

Comment:
This looks different to other profiles. Missing  <date value="2023-06-01" />
  <publisher value="NHS England" />
  <contact>
    <name value="NHS England" />
    <telecom>
      <system value="email" />
      <value value="interoperabilityteam@nhs.net" />
      <use value="work" />
      <rank value="1" />
    </telecom>
  </contact>